pimd/pim_cmd.c

```diff
@@ -1822,7 +1826,61 @@ DEFUN (show_ip_igmp_join_vrf_all,
 			first = false;
 		} else
 			vty_out(vty, "VRF: %s\n", vrf->name);
-		igmp_show_interface_join(vrf->info, vty, uj);
+		igmp_show_interface_join(vrf->info, vty, uj, GM_JOIN_STATIC);
+	}
+	if (uj)
+		vty_out(vty, "}\n");
+
+	return CMD_SUCCESS;
+}
+
+DEFUN (show_ip_igmp_proxy,
+       show_ip_igmp_proxy_cmd,
+       "show ip igmp [vrf NAME] proxy [json]",
+       SHOW_STR
+       IP_STR
+       IGMP_STR
+       VRF_CMD_HELP_STR
+       "IGMP proxy join information\n"
+       JSON_STR)
+{
+	int idx = 2;
+	bool uj = use_json(argc, argv);
+	struct vrf *vrf = pim_cmd_lookup_vrf(vty, argv, argc, &idx, uj);
+
+	if (!vrf)
+		return CMD_WARNING;
+
+	igmp_show_interface_join(vrf->info, vty, uj, GM_JOIN_PROXY);
+
+	return CMD_SUCCESS;
+}
+
+DEFUN (show_ip_igmp_proxy_vrf_all,
+       show_ip_igmp_proxy_vrf_all_cmd,
```

Comment:
Same, merge with the other show that does basically the same thing.

Reply:
same as above